### PR TITLE
dkp-toolchain-vars: add more variables to GameCube&Wii scripts

### PR DESCRIPTION
**Pull request by mardy**
Add the libogc include directory to CPPFLAGS; export the PKG_CONFIG_PATH variable to point at the portlibs pkg-config files.

### CODE REVIEW

File: dkp-toolchain-vars/cubevars.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ export PORTLIBS_CUBE=${PORTLIBS_PREFIX}
 
 export CFLAGS="-O2 -mogc -mcpu=750 -meabi -mhard-float -ffunction-sections -fdata-sections"
 export CXXFLAGS="${CFLAGS}"
-export CPPFLAGS="-D__GAMECUBE__ -I${PORTLIBS_CUBE}/include -I${PORTLIBS_PPC}/include"
+export CPPFLAGS="-D__GAMECUBE__ -I$DEVKITPRO/libogc/include -I${PORTLIBS_CUBE}/include -I${PORTLIBS_PPC}/include"
```
Comment (WinterMute) [inline dkp-toolchain-vars/cubevars.sh:9]: ${DEVKITPRO} for consistency with rest of file.

File: dkp-toolchain-vars/wiivars.sh [hunk 2]
```diff
@@ -6,7 +6,7 @@ export PORTLIBS_WII=${PORTLIBS_PREFIX}
 
 export CFLAGS="-O2 -mrvl -mcpu=750 -meabi -mhard-float -ffunction-sections -fdata-sections"
 export CXXFLAGS="${CFLAGS}"
-export CPPFLAGS="-D__WII__ -I${PORTLIBS_WII}/include -I${PORTLIBS_PPC}/include"
+export CPPFLAGS="-D__WII__ -I$DEVKITPRO/libogc/include -I${PORTLIBS_WII}/include -I${PORTLIBS_PPC}/include"
```
Comment (WinterMute) [inline dkp-toolchain-vars/wiivars.sh:9]: and again, ${DEVKITPRO}

**Review by WinterMute — CHANGES_REQUESTED** [review]

File: dkp-toolchain-vars/cubevars.sh (hunk 1 above)
Comment (mardy) [reply dkp-toolchain-vars/cubevars.sh]: Done!

File: dkp-toolchain-vars/wiivars.sh (hunk 2 above)
Comment (mardy) [reply dkp-toolchain-vars/wiivars.sh]: And done!

**Review by WinterMute — APPROVED** [review]

**Pull request merged**